src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -631,7 +630,7 @@ def test_functionapp_cors(self, resource_group):
         self.kwargs.update({
             'plan': self.create_random_name(prefix='slot-traffic-plan', length=24),
             'function': self.create_random_name(prefix='slot-traffic-web', length=24),
-            'storage': 'functioncorsstorage'
+            'storage': 'funccorsstor124'
```

Comment:
@yugangw-msft, was there a reason you changed the name?

Reply:
The existing name was taken by someone else in the world, so I had to change yesterday to avoid the conflict.
So this exposes an existing test defect, which I fixed in the latest commit .

Reply:
I see, thanks for explaining.